src/test/java/se/sundsvall/casestatus/integration/party/PartyIntegrationTest.java

```diff
@@ -55,7 +56,7 @@ void getLegalIdByPartyId_enterpriseFound() {
 	}
 
 	@Test
-	void getLegalIdByPartyId_badRequest() {
+	void getPartyIdByLegalId_badRequest() {
```

Comment:
getPartyIdByLegalId_badRequest -> getPartyIdByLegalIdBadRequest ??

(java method naming conventions doesn't allow underscore. comment applicable to other places in this PR as well)